assets.mjs

```diff
@@ -0,0 +1,44 @@
+import sharp from "npm:sharp@0.33.5";
+
+// Names of the SVGs to use for generation
+const logoFileNames = ["css", "css.square", "css.dark", "css.light"];
+
+// Start the generation process for each logo
+for (const logoFileName of logoFileNames) {
+  // Extract the type of the logo, when the type is not provided it defaults to "primary"
+  const [_, logoType = "primary"] = logoFileName.split(".");
```

Comment:
I think the doubling up of directories and partial names is somewhat confusing… I'd suggest either `/light/css.svg/` or `/css.light.svg`.

In any case I would recommend doing this in a separate step so this PR can be solely about generating the existing icons, not moving them around.

Reply:
That's fairly easy, will do makes sense.